doc/src/cylc-user-guide/suiterc.tex

```diff
@@ -615,7 +615,7 @@ \subsection{[cylc]}
     \item {\em type:} string (event handler command name or path)
     \item {\em default:} \lstinline=cylc hook check-triggering=
 \end{myitemize}
-As for any event handler, the full path can be ommited if the script is
+As for any event handler, the full path can be ommitted if the script is
```

Comment:
Getting there :laughing:, but this should actually be ``omitted``. No need to change it here though, I'll make the correction in #2910.

Reply:
Dang it! Nice catch Sadie! Thanks heaps